app/scripts/modules/core/src/managed/StatusBubbleStack.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { take } from 'lodash';
+
+import { IStatusBubbleProps, StatusBubble } from './StatusBubble';
+
+import './StatusBubbleStack.less';
+
+export interface IStatusBubbleStackProps {
+  borderColor: string;
+  maxStatusBubbleCount: number;
```

Comment:
Is there a shorter/more concise name we could give this prop? at a glance it seems a little verbose compared to other prop names nearby.